src/components/input/Switch.tsx

```diff
@@ -13,5 +17,108 @@ export default function Switch({
   disabled,
   className,
 }: SwitchProps) {
-  return <></>;
+  const [isChecked, setIsChecked] = useState(checked);
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
same here if its only for css it can just be a tailwind utility:
https://tailwindcss.com/docs/hover-focus-and-other-states